lib/matplotlib/mpl-data/stylelib/classic.mplstyle

```diff
@@ -111,8 +111,8 @@ text.usetex         : False  # use latex for all text handling. The following fo
                              # If another font is desired which can loaded using the
                              # LaTeX \usepackage command, please inquire at the
                              # matplotlib mailing list
-text.latex.unicode : False # use "ucs" and "inputenc" LaTeX packages for handling
-                           # unicode strings.
+# text.latex.unicode : False # use "ucs" and "inputenc" LaTeX packages for handling
```

Comment:
Why is this commented out? Either we leave the setting for the classic style to False (maybe we should?), or we explicitly set it to True.

Reply:
I decided to just delete them (given that the ultimate goal is to just get rid of the rcs, and that this should not change the result of any plot, just make more plots usetex-compilable).